### ISSUE TITLE
Le Startup Studio

### ISSUE THREAD

**Issue by lilobase** [scopen/scopen#4]
En france nous avons https://efounders.co/ : ils incubent le projet puis créent une structure dédiée dans laquelle ils prennent 50% des parts, les 50% restant vont à l'équipe fondatrice.

Est-ce qu'un fonctionnement similaire serait intéressant ?

**Comment by abailly** [comment]
C'est un peu à ça que je pense, moins le baratin startupesque, plus l'envie de contribuer à faire émerger des projets "utiles", même si je sais bien que ce dernier attribut peut être discuté. Colin Garriga-Salaün a essayé de faire ça avec [Yaal](https://www.yaal.fr/) mais sans grand succés semble-t-il.

**Comment by lilobase** [comment]
En dehors de l'aspect "marketing", ce que je trouvais intéressant était le mode de fonctionnement de la structure (création de filiale avec prise de participation des co-fondateurs), permettant de concilier les intérêts de l'ensemble des parties. 

Pour Yaal, les projets viennent souvent de l'extérieur de la structure donc l'approche n'est pas vraiment la même.

**Comment by abailly** [comment]
Oui, le mode de fonctionnement général me va très bien et correspond tout à fait à ce que j'avais en tête.

Pour Yaal, j'en ai discuté avec Colin une ou deux fois et effectivement, le positionnement "SSII pour startups" ne fonctionne pas bien. De manière générale, avoir une position minoritaire dans une startup ça ne fonctionne pas, sauf si on est 1/ VC et qu'on a des parts préférentiels ou 2/ employé, qu'on a des stocks et qu'on est prêt à prendre le risque d'un boulot mal payé pendant un certain temps. Et je sais de quoi je parle :-)

**Comment by damusnet** [comment]
mmmh, dire que Yaal ne fonctionne pas bien c'est très réducteur. Ils ont actuellement deux projets qui arrivent à maturité et si l'on compare au taux de mortalité "normal" des startups, c'est plutôt admirable. Evidemment pour en arriver là il a fallu beaucoup de sueur et de larmes... La difficulté dans une startup c'est de constituer l'équipe, et de concilier les motivations.

**Comment by abailly** [comment]
Je ne fais que rapporter des propos de Colin, qui date d'un an ou deux donc peut-être que les choses ont changé, ou qu'il n'était pas dans son assiette ce jour là :-) Il m'a dit faire quand même beaucoup de presta classique. Je le laisserai corriger...

**Comment by damusnet** [comment]
Oui, les choses se sont beaucoup améliorées ces douze derniers mois. Ils ne font plus de presta classique ou presque, et dernièrement ils ont même trouvé le temps d'investir de façon conséquente dans un très beau projet interne.

**Comment by abailly** [comment]
Et bien c'est une excellente nouvelle. Et qui donne espoir :-)
